packages/react/docs/guides/onboarding.md

```diff
@@ -0,0 +1,259 @@
+we can do so without lots of code updates.
+
+```jsx
+// importing
+import { settings } from '<react-package>/src/constants/Settings.js';
+
+// prefix is used for Carbon, iotPrefix for our components
+const { prefix, iotPrefix } = settings;
+
+// usage
+<Button className={`${iotPrefix}--button`}>A prefixed button</Button>;
+```
+
+This same structure is available within the scss files, too.
+
+```scss
+// this is using @use from newer versions of sass, but isn't supported until v3 of this library
+// is released in 2022. The method in v2 is the usual @import
+@use '<react-package>/src/globals/vars' as *;
+
+// for v2
+// @import '<react-package>/src/globals/vars';
+
+.#{$prefix}--button {
+  // styles to override the carbon button
+}
+
+.#{$iot-prefix}--button {
+  // styles to apply to the iot button
+}
+```
+
+### Using Carbon tokens
+
+Instead of hard-coding many values, we rely on tokens from Carbon to make theming and re-use
+easier. There are a multitude of tokens that can be used in a a variety of situations and themes. We
+use the Gray 10 theme in PAL, and you can find more details about Carbon tokens in their documentation.
+
+- Color: https://www.carbondesignsystem.com/guidelines/color/usage/
+- Motion: https://www.carbondesignsystem.com/guidelines/motion/usage/
+- Spacing: https://www.carbondesignsystem.com/guidelines/spacing/overview/
+- Themes: https://www.carbondesignsystem.com/guidelines/themes/overview/
+- Typography: https://www.carbondesignsystem.com/guidelines/typography/overview/
+- Icons: https://www.carbondesignsystem.com/guidelines/icons/library/
+
+```scss
+// Bad example
+button {
+  padding: 1rem;
+  color: black;
+}
+
+// Good example, v3 @use
+@use '<react-package>/src/globals/spacing' as *;
+@use '<react-package>/src/globals/colors' as *;
+
+// v2 @import
+// @import '<react-package>/src/globals/vars';
+// @import '<react-package>/src/globals/spacing';
+
+button {
+  padding: $spacing-05;
+  color: $text-01;
+}
+```
+
+### BEM naming convention
+
+When crafting css classes we use a BEM (Block, Element, Modifier) syntax to help make following the
+styles easier. We don't use an exact BEM format because of the prefix mentioned above, so you'll always
+want to start class with a prefix. The structure should look like this: `prefix--block__element--modifier`
+or `prefix--block--modifier`.
+
+- Prefix: A constant imported from settings to namespace this component from other Carbon or third-party components.
+- Block: A standalone entity that is meaningful on its own.
+- Element: An element is a part of a block that has no standalone meaning and is semantically tied to its block.
+- Modifier: A flag on a block or element. Use them to change appearance or behavior.
+
+You can learn more about BEM in this [documentation](http://getbem.com/introduction/).
+
+```scss
+// v3 @use
+@use '<react-package>/src/globals/vars' as *;
+@use '<react-package>/src/globals/spacing' as *;
```

Comment:
comment this out for now instead of the v2 import since we are still mainly developing on v2?

Reply:
Yep. I've changed everything to v2 being uncommented.